src/components/Button/Button.tsx

```diff
@@ -1,15 +1,14 @@
-import React, { ReactNode } from 'react'
+import React, { ReactNode, FC } from 'react'
 
 interface IButtonProps {
-  name?: string,
   className?: string,
   disabled: false,
   children: ReactNode 
 }
 
-const Button = ({ className, disabled, name }: IButtonProps) => {
+const Button: FC<IButtonProps> = ({ className, disabled, children}) => {
```

Comment:
Да, но как будто не отформатировал.

Reply:
Слева есть пробел, а справа нет `¯\_(ツ)_/¯ `
```suggestion
const Button: FC<IButtonProps> = ({ className, disabled, children }) => {
```